src/main/java/frc/trigon/robot/subsystems/swerve/swervemodule/SwerveModule.java

```diff
@@ -11,144 +11,148 @@
 import edu.wpi.first.wpilibj.sysid.SysIdRoutineLog;
 import frc.trigon.robot.constants.RobotConstants;
 import frc.trigon.robot.poseestimation.poseestimator.PoseEstimatorConstants;
+import frc.trigon.robot.subsystems.swerve.SwerveConstants;
 import org.trigon.hardware.phoenix6.cancoder.CANcoderEncoder;
 import org.trigon.hardware.phoenix6.cancoder.CANcoderSignal;
 import org.trigon.hardware.phoenix6.talonfx.TalonFXMotor;
 import org.trigon.hardware.phoenix6.talonfx.TalonFXSignal;
 import org.trigon.utilities.Conversions;
 
 public class SwerveModule {
-    private final TalonFXMotor driveMotor, steerMotor;
+    private final TalonFXMotor
+            driveMotor,
+            steerMotor;
     private final CANcoderEncoder steerEncoder;
     private final double wheelDiameterMeters;
     private final PositionVoltage steerPositionRequest = new PositionVoltage(0).withEnableFOC(SwerveModuleConstants.ENABLE_FOC);
     private final VelocityTorqueCurrentFOC driveVelocityRequest = new VelocityTorqueCurrentFOC(0);
     private final VoltageOut driveVoltageRequest = new VoltageOut(0);
     private final TorqueCurrentFOC driveTorqueCurrentFOCRequest = new TorqueCurrentFOC(0);
-    private boolean driveMotorClosedLoop = false;
-    private double[]
-            latestOdometryDrivePositions = new double[0],
-            latestOdometrySteerPositions = new double[0];
+    private boolean shouldDriveMotorUseClosedLoop = false;
     private SwerveModuleState targetState = new SwerveModuleState();
+    private double[]
+            latestOdometryDrivePositions,
+            latestOdometrySteerPositions;
 
-    public SwerveModule(int moduleID, double offsetRotations, double wheelDiameterMeters) {
+    public SwerveModule(int moduleID, double wheelDiameterMeters, double offsetRotations) {
+        this.wheelDiameterMeters = wheelDiameterMeters;
```

Comment:
Put this after all uses of `moduleID`

Reply:
Why? I know you prefer to use the params in the order their given, but I prefer to write any "this..." first. Deal with it.

Reply:
You can do both...

Reply:
No, because I also want the ID to be the first param.